### PR TITLE
Show the Giving Block api call failure errors before moving to Checkout Screen

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4547

image

### CODE REVIEW

File: components/contribution-flow/StepCheckout.js [hunk 1]
```diff
@@ -47,25 +46,17 @@ const StepCheckout = ({ stepDetails, order }) => {
               includeMargin
             />
             <P mb="16px">{order.paymentMethod.data.depositAddress}</P>
+            <StyledButton onClick={() => copy(window.location.href)} disabled={isCopied}>
```
Comment (znarf) [inline components/contribution-flow/StepCheckout.js:49]: Copying `window.location.href`? What is this?

Comment (znarf) [reply components/contribution-flow/StepCheckout.js:49]: "Click to copy wallet address"

Comment (SudharakaP) [reply components/contribution-flow/StepCheckout.js]: My mistake. This should be copying the wallet address. I've corrected that. 👍🏽

**Review by znarf — APPROVED** [review]
Ok for me. Not tested.

**Comment by SudharakaP** [comment]
> Ok for me. Not tested.

Thanks a lot for the reviews. I'll do some more testing and merge it today.

**Pull request merged**